src/testing/test-transpile.ts

```diff
@@ -10,12 +10,12 @@ export function transpile(input: string, opts: TranspileOptions = {}): Transpile
     coreImportPath: isString(opts.coreImportPath) ? opts.coreImportPath : '@stencil/core/internal/testing',
     currentDirectory: opts.currentDirectory || process.cwd(),
     module: 'cjs', // always use commonjs since we're in a node environment
-    proxy: null,
+    proxy: undefined,
```

Comment:
this too!

https://github.com/ionic-team/stencil/blob/fa074ad852bda39ccd0a1c5f983cf16735593e82/src/declarations/stencil-public-compiler.ts#L2696-L2701